src/main/java/io/buildpacks/example/sample/CalculatorService.java

```diff
@@ -2,25 +2,63 @@
 
 import org.springframework.stereotype.Service;
 
+/**
```

Comment:
4. **Unit tests**: It seems like you have implemented a basic calculator service. It would be beneficial to write unit tests to cover each method to ensure they work as expected.